tracer/src/Datadog.Trace/Configuration/ImmutableTracerSettings.cs

```diff
@@ -38,6 +38,7 @@ public partial record ImmutableTracerSettings
         private readonly IReadOnlyDictionary<string, string> _globalTags;
         private readonly double? _globalSamplingRate;
         private readonly bool _runtimeMetricsEnabled;
+        private readonly string? _customSamplingRulesInternal;
```

Comment:
nit: we don't need the `Internal` suffix + shall we maybe rename this to `_localCustomSamplingRues`?

Reply:
None of the other settings that have remote overrides use `local` in the name, so I'll rename to `_customSamplingRules` for consistency.

Reply:
renamed